### ISSUE TITLE
Firmware upgrade for Windows

### ISSUE THREAD

**Issue by nomis52** [OpenLightingProject/ja-rule#169]
We need to figure out and document how to upgrade the firmware from Windows.

These links may be helpful:

http://wiki.openmoko.org/wiki/Dfu-util-windows
https://github.com/libusb/libusb/wiki/Windows
https://community.particle.io/t/tutorial-installing-dfu-driver-on-windows-24-feb-2015/3518

**Comment by peternewman** [comment]
The bottom link works fine on Win 7.

**Comment by peternewman** [comment]
Although we then need to work out what interface is what and load some sensible drivers to make them useable.

**Comment by nomis52** [comment]
@peternewman do you mean you've actually loaded f/w using the last link?

**Comment by peternewman** [comment]
Correct, it throws an error about not being able to disconnect or something, but I only flashed the bootloader via ICSP and did everything else with DFU. I'll do something more serious later like change the device name, just to prove DFU is really working. Also given it's libusb, I think USB on Windows should just work too!
